posthog/migrations/0166_feature_flag_override.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.1.12 on 2021-09-06 18:59
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0165_dashboarditem_dive_dashboard"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="FeatureFlagOverride",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("override_value", models.JSONField(default=bool)),
```

Comment:
What does `default=bool` do for a JSONField? Will `True` be the default `bool`? Or `False`? I get `default=dict` to initialize `{}`, but this is not very explicit... 🤔

Reply:
Good catch! The override should be explicitly set each time, so I just removed the default.

FWIW, while definitely bad, `default=bool` gives you `False` because `bool()` evaluates to `False`. With JSON fields, you can't just give the default value (`default=False`). It has to be a callable. I think it has to do with the way Postgres stores JSON fields, and multiple rows potentially referencing the same instance of the JSON object.


The more explicit/better way of doing it is probably something like:
```
def get_false():
    return False

class FeatureFlagOverride(models.Model):
    override_value: models.JSONField = models.JSONField(default=get_false)
```